invisu.csl

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded" default-locale="fr-FR">
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="expanded">
   <info>
-    <title>InVisu (French)</title>
+    <title>InVisu (International)</title>
```

Comment:
Could you delete the " (International)" from the style title? We have quite a few more styles without a `default-locale`, and none are currently tagged as such, and I'd like to keep things consistent.